tex/alg-NT/dedekind.tex

```diff
@@ -526,6 +536,13 @@ \section{Fractional ideals}
 			\mid x\ka \subseteq (1) = \mathcal A \right\} \]
 	is a fractional ideal with $\ka \ka\inv = (1)$.
 \end{theorem}
+
+(This result is nontrivial.
```

Comment:
![image](https://github.com/vEnhance/napkin/assets/25191436/50573748-be97-41d1-9864-f7d7796f69df)

In retrospect, it's already mentioned somewhere else in the book.